README.md

```diff
@@ -16,6 +16,11 @@ limitations under the License.
 {% endcomment %}
 -->
 
+
+[![PyPI version](https://badge.fury.io/py/elyra.svg)](https://badge.fury.io/py/elyra)
+[![GitHub](https://img.shields.io/badge/issue_tracking-github-blue.svg)](https://github.com/elyra-ai/elyra/issues)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/elyra-ai/elyra/master?urlpath=lab/)
+
```

Comment:
These look good.  Could we also add a [workflow badge](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository) for our "build" actions?

The downloads badge is kinda cool as well.

Reply:
added downloads badge, will add workflows bade when we have a daily build workflow on our master